static/templates/stream_settings/stream_settings_checkbox.hbs

```diff
@@ -21,7 +21,7 @@
     </p>
     {{/if}}
     {{#if (eq setting_name "push_notifications")}}
-    <i class="fa fa-question-circle settings-info-icon {{#unless disabled_realm_setting}}hide{{/unless}} tippy-zulip-tooltip"
+    <i class="fa fa-exclamation-circle settings-info-icon {{#unless disabled_realm_setting}}hide{{/unless}} tippy-zulip-tooltip"
```

Comment:
Is this change discussed somewhere? We still use `fa-info-circle` in settings. And we probably would have to instead change the tooltip to show when hovering over the input or lable instead of having a icon like we did recently for `Deactivate organization` button. Till then we can probably use `fa-info-circle` for consistency.

Reply:
This was discussed in https://github.com/zulip/zulip/pull/23943#issuecomment-1409161116